docs/tutorials/nb_distribution_statistics.ipynb

```diff
@@ -17,12 +17,14 @@
     "\n",
     "In order to do that, you can use statistical tests. In this tutorial we present how to easily do this using the `DistributionStatistics` class, and with the statistical tests directly.\n",
     "\n",
-    "The available statistical tests in `probatus.stat_tests` are: \n",
-    "- Epps-Singleton ('ES')\n",
-    "- Kolmogorov-Smirnov statistic ('KS')\n",
-    "- Population Stability Index ('PSI')\n",
-    "- Shapiro-Wilk based difference statistic ('SW')\n",
-    "- Anderson-Darling TS ('AD')\n",
+    "Available tests:\n",
```

Comment:
![image](https://user-images.githubusercontent.com/40659437/129701594-22e162ce-a922-48b0-8d44-4c21b110d852.png)
Would it be possible to have at the top the AutoDist and Distribution statistic, since these are the most timportant ones, and then group all the tests into one subgroup below?

Also no need to expose utils docs.

Reply:
FIxed
![image](https://user-images.githubusercontent.com/16015401/129712607-3e146cea-e405-41aa-b496-d4c63438f229.png)